lxqt-config-session/translations/lxqt-config-session_el_GR.ts

```diff
@@ -1,35 +1,35 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version='1.0' encoding='utf-8'?>
 <!DOCTYPE TS>
 <TS version="2.1" language="el_GR">
 <context>
     <name>AutoStartEdit</name>
     <message>
-        <location filename="../autostartedit.ui" line="14"/>
+        <location line="14" filename="../autostartedit.ui"/>
```

Comment:
There are many unnecessary changes that logicaly mean noop (e.g. here just attributes reordering). Can you, please, get rid of them?